app/celery/send_va_profile_notification_status_tasks.py

```diff
@@ -0,0 +1,74 @@
+import requests
+
+from flask import current_app
+
+from app import notify_celery, va_profile_client
+from app.celery.exceptions import AutoRetryException
+from app.constants import DATETIME_FORMAT, EMAIL_TYPE
+from app.feature_flags import FeatureFlag, is_feature_enabled
+from app.models import Notification
+
+
+def check_and_queue_va_profile_notification_status_callback(notification: Notification) -> None:
+    """
+    This checks the feature flag to send SMS statuses is enabled.
+    If it is, queues the celery task and collects data from the notification. Otherwise, it only logs a message.
+
+    Status for email notifications are always sent.
+
+    :param notification: the notification (email or sms) to collect data from
+    """
+    current_app.logger.debug(
+        'Sending notification status to VA Profile, checking VA_PROFILE_SMS_STATUS_ENABLE feature flag... | notification %s',
+        notification.id,
+    )
+
+    if is_feature_enabled(FeatureFlag.VA_PROFILE_SMS_STATUS_ENABLED) or notification.notification_type == EMAIL_TYPE:
+        current_app.logger.debug(
+            'Sending notification status to VA Profile, collecting data for notification %s', notification.id
+        )
+
+        notification_data = {
+            'id': str(notification.id),  # this is the notification id
+            'reference': notification.client_reference,
+            'to': notification.to,  # this is the recipient's contact info
+            'status': notification.status,  # this will specify the delivery status of the notification
+            'status_reason': notification.status_reason,  # populated if there's additional context on the delivery status
+            'created_at': notification.created_at.strftime(DATETIME_FORMAT),  # noqa: F821
```

Comment:
I don't think this `noqa` is necessary here, is it?

Reply:
Updated.